content/applications/hr/recruitment/source_analysis.rst

```diff
@@ -0,0 +1,78 @@
+:guilabel:`Hired`). Hover the cursor over any column to view the specific numbers fort that column.
+
+.. image:: source_analysis/source-analysis.png
+   :align: center
+   :alt: The default bar chart of the source analysis information.
+
+Pivot Table
+===========
+
+To view more details, view the :guilabel:`Source Analysis` report in a pivot table. While on the
+:guilabel:`Source Analysis` report page, click the :icon:`oi-view-pivot` :guilabel:`Pivot` button in
+the top-right corner. The data is presented in a pivot table, with rows populated by job position,
+and columns populated stage.
+
+To expand this chart to show what specific sources the applicants came from, click into the
+:icon:`fa-plus-square` :guilabel:`Total` box, above the columns to reveal a drop-down menu, and
+click :guilabel:`Source`.
+
+Now, each column is grouped by the source: :guilabel:`Search engine`, :guilabel:`Facebook`,
+:guilabel:`Newsletter`, etc. Each source displays the separate count for :guilabel:`Applicant`,
+:guilabel:`Hired`, and :guilabel:`Refused`, for each source.
+
+This information as presented makes it difficult to view the specific numbers for each source. Click
+the :icon:`fa-exchange` :guilabel:`Flip axis` icon, in the upper-left corner of the page, to swap
+the information. After clicking the :icon:`fa-exchange` :guilabel:`Flip axis` icon, the rows
+represent the source, and the columns represent the job positions, further divided by stage.
+
+.. image:: source_analysis/source-pivot.png
+   :align: center
+   :alt: The axes flipped in the source analysis report, in pivot table view.
+
+In this view, the total number of applicants, hired employees, and refused applicants, are displayed
+for each source, as well as for each stage by job position.
+
+Medium
+------
+
+Sometimes, further information is desired to gain a more complete insight as to the complete source
+of applicants. Viewing the medium for the applicants can be beneficial to see which specific medium
+is more successful.
+
+To further group the results by medium, click into a row beneath the :icon:`fa-minus-square-o`
+:guilabel:`Total` box. Click :guilabel:`Medium` in the resulting drop-down menu. The row presents
+the specific mediums, relevant to that specific source.
+
+Once :guilabel:`Medium` is selected for one source, clicking into another row automatically reveals
+the specific metrics for the mediums for that source.
+
+.. image:: source_analysis/medium.png
+   :align: center
+   :alt: The sources rows, expanded to also show the medium for each source.
+
+.. note::
+   The only mediums that appear for a source, are mediums that have been selected on an applicant's
+   form. If a medium has **not** been selected for any applicants, the medium **does not appear** in
+   the drop-down rows beneath the source.
+
+   For example, if no applicant's selected the medium :guilabel:`Google Adwords`, that medium does
+   **not** appear beneath the :guilabel:`Search engine` source row.
```

Comment:
This document has enough as is, but since I'm reviewing, I just wanted to help brainstorm some ideas for other use cases you can add to this doc in a follow-up PR for 2 points :blush: 

1. Candidate engagement report: Analyzing where candidates drop out of the application process for each source.
  Idea to generate this report (feel free to explore and poke around to improve this! I was just having fun exploring the report you introduced me to :smile:)
  
![image](https://github.com/user-attachments/assets/2eda863f-5b83-4a0a-8be0-c1cf4f565c0a)

(I'm not sure how to do this, as it looks like we can't get the stage names from each process to appear. Only "In progress/Hired. Maybe you'll figure out a workaround?)

2. Time to Hire report: Measuring how long it takes to fill a position from each source. 
  Not super sure how to make this appear, but there's a "Process Duration" measure. Maybe you can look more into this? There's also a "# of Meetings" measure!
  
![image](https://github.com/user-attachments/assets/05bec4f5-b42d-4321-8e87-087d7c426095)

3. Referral success rate: analyze how much better candidates with referrals perform compared to regular hires
  using the measures (:

4. Application to hire ratio:  Understanding how many applications from each source result in a hire.
 just the measure in the report (easiest one!)

Reply:
THANK YOU!!! I have SUCH a hard time trying to make these reports give me the information I a looking for. I use chat gpt, I play around with the reporting measures and features, and I still get stuck. It's funny, I know what kind of information I want to see, but I cannot figure out how to get that information to be shown =D 

I will absolutely expand on this! I have another recruitment reporting doc being written now, but also wrestling with how the information is gathered and shown. I would love for you to review that doc as well since you did such a great job with this one!